packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -43,7 +43,7 @@ function Field(props: SpectrumFieldProps, ref: RefObject<HTMLElement>) {
     labelProps,
     readOnlyText,
     isReadOnly,
-    inputProps, 
+    readOnlyInputProps, // any better names to call this?
```

Comment:
what is the need for this?

Reply:
Picker doesn't have inputProps so eventually those have to get created in ReadOnlyField. But both textField and textArea pass it down, so to avoid doing useTextField twice for something that already exists, I renamed inputProps to readOnlyInputProps so that I could distinguish between something that was passed from another component vs something that needed to be created since in ReadOnlyField I have `let {inputProps} = useTextField()`. 

Actually, now that I'm looking back on my code, I don't think it's working the way I want it to but that was at least the thought process behind it. Open to changing/revising it though since I was a little unsure of this myself

Reply:
inputProps already goes through correctly, so I think we can just make a variable in Picker.tsx called inputProps that we pass to Field whenever isReadOnly
Would that work?